.github/workflows/main.yml

```
@@ -0,0 +1,38 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    container:
+      image: elementary/docker:unstable
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install Dependencies
+      run: |
+        apt update
+        apt install -y libfolks-dev libglib2.0-dev libgranite-dev libgtk-3-dev
+        apt install -y meson valac
```

```suggestion
        apt install -y libfolks-dev
```

Actually the elementary docker installs fundamental dependencies like meson or valac, so what we need to explicit is only folks

---

are glib, granite and gtk needed or are they also installed by the elementary docker image?

---

The elementary docker image [installs `elementary-sdk`](https://github.com/elementary/docker/blob/master/odin-unstable/Dockerfile#L13) which [includes glib, granite, gtk, meson, and valac as dependencies](https://github.com/elementary/metapackages/blob/master/elementary-sdk-amd64).

---

Done.